Move to previous next parent (#1167)
VirtusLab/git-machete-intellij-plugin: CHANGE-NOTES.md

## Unreleased

## v3.1.0

> - Added branch navigation allowing to checking out the next, previous, and parent branches in the machete layout, with `ctrl+alt+shift+G`, `ctrl+alt+shift+B`, and `ctrl+alt+shift+U`, respectively.

Reviewer: Pls update accordingly

> 

Reviewer: Pls also update  docs/features.md (no need to add screenshots, just text is ok)

> - Added branch navigation allowing to checking out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` in the machete layout.

Reviewer: ```suggestion
- Added branch navigation allowing to checking out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened.
```

## v3.1.0
- Added support for IntelliJ 2022.3.

> - Added branch navigation allowing to check out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened, and has focus.

Reviewer 2: no yelling Xd I am just excited b/c we are so close to merge this one

> - Added branch navigation allowing to check out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened, and has focus.

Author: ok, so doing it right now.

> - Added branch navigation allowing to check out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened, and has focus. Note that `option` key is an `alt` key counterpart in macOS.

Reviewer: ```suggestion
- Added branch navigation allowing to check out:
  * previous branch with `alt + up-arrow`,
  * next branch with `alt + down-arrow`,
  * parent branch with `alt + left-arrow`,
  * first child branch `alt + right-arrow`
  when Git Machete tab is opened, and has focus. Note that `option` key is an `alt` key counterpart in macOS.
```

> - Added branch navigation allowing to check out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened, and has focus. You would need to use the `option` key instead of `alt` in MacOS.

Author: @mkondratek please check 👍

> - Added branch navigation allowing to check out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened, and has focus. You would need to use the `option` key instead of `alt` in MacOS.

Reviewer 2: ```suggestion
- Added branch navigation allowing to check out the next branch with `alt + down-arrow` or `alt + right-arrow`, previous branch with `alt + up-arrow`, and parent branch with `alt + left-arrow` when Git Machete tab is opened, and has focus. Note that `option` key is an `alt` key counterpart in macOS.
```
